### ISSUE TITLE
wrong ordering in records returned by records() iterator

### ISSUE THREAD

**Issue by valerino** [omerbenamram/pyevtx-rs#25]
hi, i think there is a bug in event parsing regarding ordering.
the PyEvtxParser().records() iterator return records parsing each chunk in DESCENDING order instead of ASCENDING, so when iterating chunks you obtain i.e.

(iterating chunk0): record10,record9,record8,record7,record6,record5,record4,record3,record2,record1
(iterating chunk1)chunk1: record20,record19,record18,record17,record16,record15,record14,record13,record12,record11
and so on ...

basically, seems records in each chunk are orderered in a descending way before being returned by the iterator, and this leads to the records not being in the original order. this may break some utilization of your lib where the original order needs to be preserved.
